packages/form-config/src/components/Releases/index.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { useEffect, useState, ChangeEvent } from 'react';
```

Comment:
The interface of the access token here should also factor we can pass as promise. Check [here](https://github.com/OpenSRP/web/blob/9c5aa298a52b0adf189d4c4788262695f26cc0c0/packages/opensrp-server-service/src/serviceClass.ts#L110)